src/sentry/plugins/sentry_mail/activity/assigned.py

```diff
@@ -12,25 +12,36 @@ def get_activity_name(self):
         return 'Assigned'
 
     def get_description(self):
-        # TODO(mattrobenolt): Handle when assignee is Team
-
         activity = self.activity
         data = activity.data
-        if activity.user_id and six.text_type(activity.user_id) == data['assignee']:
-            return u'{author} assigned {an issue} to themselves'
-
-        try:
-            assignee = User.objects.get_from_cache(id=data['assignee'])
-        except User.DoesNotExist:
-            pass
-        else:
-            return u'{author} assigned {an issue} to {assignee}', {
-                'assignee': assignee.get_display_name(),
-            }
-
-        if data.get('assigneeEmail'):
-            return u'{author} assigned {an issue} to {assignee}', {
-                'assignee': data['assigneeEmail'],
-            }
-
-        return u'{author} assigned {an issue} to an unknown user'
+        if data['assigneeType'] == 'user':
```

Comment:
We could/should change this to be `if 'assigneeType' not in data or data['assigneeType'] == 'user'`.

Just for some possibly backwards compatibility between old style Activity objects and new ones.